vuepress/docs/.vuepress/next.js

```diff
@@ -1,69 +1,129 @@
 module.exports = {
     docsSidebar: function(path) {
         return [
-            {
-                title: 'Introduction',
-                path: path,
-            },
             {
                 title: 'Getting Started',
```

Comment:
I feel like making the section content display by default is more usable, as we discussed. It also simplifies the left nav a bit. I tried this for the docs in my local:
```
               title: 'Overview',
                path: path,
                children: [
                    {
                        title: 'Get Started',
                        path: path + 'getting-started/',
                    },
                    {
                        title: 'Concepts',
                        path: path + 'getting-started/concepts-overview.md',
                    },
                ]
 ```

I think we want the same thing for the tutorials.

What I'm uncertain about is where we explain the Create-Curate-Compose-Consume idea in a way that it will get seen. We might need it at the top-level of docs and tutorials page, for docs definitely ahead of the diagram explaining the Entando architecture. We want to explain what Entando accomplishes. rather than how it does it, at least for the first impression.